daisy_workflows/linux_common/bootstrap.sh

```diff
@@ -131,4 +131,11 @@ chmod +x "$path_script"
 CheckPython3Installation
 
 logger -p daemon.info "Status: Running $path_script"
+# Remove the trap when script_prints_status equals "yes".
+# Useful when the script handles its own communication with
+# daisy (typically with prefixes such as "TranslateSuccess"
+# and "TranslateFailure").
+GetMetadataAttribute "script_prints_status"
```

Comment:
it looks like "bootstrap.sh" is the only place to use this mechanism, however the attr name "script_prints_status" is very general.

Reply:
The term "script" references the script that `boostrap.sh` will be executing. This is what it looks like in a workflow file: 

![image](https://user-images.githubusercontent.com/575626/99120566-eec00c00-25af-11eb-861f-e9f0dea0afcc.png)

Reply:
In other words, it's an attribute of the parameter `script`, rather than `bootstrap.sh`.

Reply:
Cool